packages/shared/components/popups/ActivityDetailsPopup.svelte

```diff
@@ -192,7 +192,7 @@
             <ActivityInformation {activity} />
         </alias-details>
     {:else if activity?.type === ActivityType.Nft}
-        <NftActivityDetails {...details} />
+        <NftActivityDetails activity={{ ...details }} />
```

Comment:
please use the same approach as with the `Alias`:
```suggestion
        <nft-details class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0">
            <NftActivityDetails {activity} />
            <ActivityInformation {activity} />
        </nft-details>
```

Reply:
+ remove line 101 - 109